sdk/monitor/azure-monitor-ingestion/samples/async_samples/sample_upload_file_contents_async.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
Missing copyright header: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/sample_analyze_business_cards.py#L3

Also consider adopting the sample header description block in the link

Reply:
Thanks for the pointer. Went ahead and updated all the samples with a copyright header and more detailed description block.